### ISSUE TITLE
[Fabric 1.19.2] Incompatibility with Snail Mail Mod

### ISSUE THREAD

**Issue by whatamaniac** [architectury/architectury#437]
Makes the Mod Snail Mail Unplayable with the latest version of Architectury API on Fabric 1.19.2 as there's a UI Element/Screen Missing where you register a name to your mailbox and select which mailbox to send the mail to.

**Potential Fix:** Downgrading Architectury dependency to 6.4.62 fixes this issue, at least for minecraft 1.19.2 but Unfortunately downgrading by such a huge jump breaks lots of other mods which are dependent on Architectury.

![261903064-610508f1-f1c5-422f-93a7-35aac39ebf9a](https://github.com/architectury/architectury-api/assets/75019293/6b2ce14c-d551-4e77-b86e-5f83dde7ae1c)

https://github.com/itsmeowCommissions/SnailMail/issues/18

Please let us know if there are any plans on fixing this in the Next Update.. If not.. Lemme know if i could fix this temporarily myself in some way or the other?

**Comment by ChiefArug** [comment]
Your version of architectury is very outdated. The mods you have require [a much newer version](https://www.curseforge.com/minecraft/mc-mods/architectury-api/files/4555748).

Also, remove optiglue. It is for 1.19.3+ but you are playing 1.19.2.

**Comment by ChiefArug** [comment]
As for that issue you linked, you'll need to wait for the author of that mod to fix the issue, or find the actual bug causing it in architectury (its probably an issue with the mod though).

**Comment by whatamaniac** [comment]
> Your version of architectury is very outdated. The mods you have require [a much newer version](https://www.curseforge.com/minecraft/mc-mods/architectury-api/files/4555748).
> 
> Also, remove optiglue. It is for 1.19.3+ but you are playing 1.19.2.

Yup i've fixed that... using the latest version of Architectury and removed Optiglue.

**Comment by itsmeow** [comment]
The issue was because Architectury moved the trigger location of the Block Place event between 6.4.62 and 6.4.63. Can be closed now because I have fixed the issue in Snail Mail.
